tmt/tests/bootc-install-provision.sh

```diff
@@ -91,7 +95,51 @@ COPY bin /usr/local/bin
 
 # In Testing Farm, all ssh things should be reserved for ssh command run after reboot
 COPY .ssh /var/roothome/.ssh
+COMMONEOF
+
+    if [[ -v KOJI_TASK_ID ]] || [[ -v CI_KOJI_TASK_ID ]]; then
```

Comment:
Shouldn't these have a `$`?

Reply:
I double checked from google, `-v` needs variable name, do not need `$`.

Reply:
Ah right, I was thinking this was `test -n`. I hadn't seen much use of `-v` before.